### PR TITLE
Rewrite v2/v3

### PR DESCRIPTION
**Pull request by Hackante**

### CODE REVIEW

File: commands/game/bomb.js [hunk 1]
```diff
@@ -1,67 +1,122 @@
...
+        let msg = await message.channel.send({ content: "Place your bombs!", components: createPlayerComponents(players) })
+
+        msg.awaitMessageComponent({ filter: (i) => i.user.id === message.author.id })
+        .then(async i => {
+            let gPhase = db.get(`gamePhase`)
+            night = Math.floor(gamePhase / 3) + 1
+            if (gPhase % 3 !== 0) return i.update({ content: "Place your bombs!", components: [] })
+            let row = Math.ceil(i.customId.slice(1)/4)
+            let column = setup[row-1].indexOf(i.customId.slice(1)) + 1
+            let newComponents = createPlayerComponents(players)
```
Comment (Hackante) [inline commands/game/bomb.js:72]: returns `void` see other comment

**Review by Hackante — COMMENTED** [review]
We should probably also delete all variables that are not being used in the command files but there were to many for me to comment them

**Comment by Hackante** [comment]
@3061LRTAGSPKJMORMRT a few points I wanted to mention. I hope you are able to commit the changes too and fix errors. Thanks for the work 🙏🏻

**Comment by 3061LRTAGSPKJMORMRT** [comment]
Alright, I have fixed those errors locally, but I will commit them once the bot is fully functional

**Comment by 3061LRTAGSPKJMORMRT** [comment]
The code above has been tested and should work. If there are bugs, it would be minor and not something that will disrupt the bot

**Comment by Hackante** [comment]
Don't delete the branch yet. Just in case

**Pull request merged**